easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -809,6 +809,10 @@ def validate_iterate_opts_lists(self):
         Configure/build/install options specified as lists should have same length.
         """
 
+        # Disable templating as only the existance is of interest, the actual resolved value is not
+        prev_enable_templating = self.enable_templating
+        self.enable_templating = False
```

Comment:
Why would you change that? This flag is precisely for such use-cases isn't it?

This would change the semantic of `get_ref` considerably. If you want to avoid the `enable_templating`, I'd rather add `resolve=True` as a parameter to `get`

Reply:
Ah, I see the reasoning. Changed `get`